src/main/java/ch/njol/skript/effects/EffDrop.java

```diff
@@ -41,69 +30,65 @@
 import ch.njol.skript.util.Direction;
 import ch.njol.skript.util.Experience;
 import ch.njol.util.Kleenean;
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.ExperienceOrb;
+import org.bukkit.entity.Item;
+import org.bukkit.event.Event;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.util.Vector;
+import org.eclipse.jdt.annotation.Nullable;
 
-/**
- * @author Peter Güttinger
- */
 @Name("Drop")
 @Description("Drops one or more items.")
 @Examples({"on death of creeper:",
 		"	drop 1 TNT"})
 @Since("1.0")
 public class EffDrop extends Effect {
+
 	static {
-		Skript.registerEffect(EffDrop.class, "drop %itemtypes/experiences% [%directions% %locations%]",
-				"drop %itemtypes/experiences% [%directions% %locations%] without velocity");
+		Skript.registerEffect(EffDrop.class, "drop %itemtypes/experiences% [%directions% %locations%] [(1¦without velocity)]");
 	}
-	
+
 	@Nullable
 	public static Entity lastSpawned = null;
-	
-	@SuppressWarnings("null")
+
+	@SuppressWarnings("NotNullFieldNotInitialized")
 	private Expression<?> drops;
-	@SuppressWarnings("null")
+	@SuppressWarnings("NotNullFieldNotInitialized")
 	private Expression<Location> locations;
-	
-	private boolean useVelocity = true;
-	
-	@SuppressWarnings({"unchecked", "null"})
+
+	private boolean useVelocity;
+
+	@SuppressWarnings("unchecked")
```

Comment:
This should come after Override (according to code conventions, although they probably could use some updating in places)